recipes/r-fastcpd/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+${R} --slave -e "install.packages('fastglm', repos = 'https://cloud.r-project.org')"
```

Comment:
This is not allowed. It results in the build here interpreting everything that gets side-loaded as part of this package. That is, it will lead to clobbering later on.

If this needs `fastglm`, then please create or request a recipe for `r-fastglm` be added.